docs/appendix/zowe-yaml-configuration.md

```diff
@@ -516,11 +527,13 @@ These configurations can be used under the `components.gateway` section:
 - **apiml.security.oidc.registry**  
   Specifies the SAF registry used to group identities recognized as having an OIDC identity mapping. The registry name is the string used during the creation of the mapping between distributed and mainframe user identities. For more information, see [ESM configuration prerequisites](../extend/extend-apiml/api-mediation-oidc-authentication.md#esm-configuration-prerequisites).
 - **apiml.security.oidc.jwks.uri**  
-  Specifies the URI obtained from the authorization server's metadata where the Gateway queries for the JWK used to sign and verify the access tokens. A valid value is any valid URI.
+  Specifies the URI obtained from the authorization server's metadata where the Gateway queries for the JWK used to sign and verify the access tokens. A valid value is any valid URI. Starting from Zowe version 3.4.0 and later versions, this parameter will support one or more URIs (comma-separated). This allows the Gateway to support multiple OIDC providers at the same time.
```

Comment:
In the sentence beginning with "This allows the Gateway to support..." please specify what is "This". Does "this" mean specifying this parameter as 'true'?

Reply:
I can rephrase it to something like "Providing multiple JWK URIs allows Gateway to support multiple OIDC providers at the same time". How does it look?

Reply:
Your revised language sounds good. Thanks